frontend/src/components/TimeEntry/SelectHoursComponent.js

```diff
@@ -26,10 +26,10 @@ const SelectHoursComponent = ({numberOfHoursForDay, entryDuration}) => {
     if(entryDuration){
       return parseInt(hour) + (parseInt(numberOfHoursForDay) - entryDuration) > 8;
     }
-    return parseInt(hour) + parseInt(numberOfHoursForDay) > 8;
+    return parseInt(hour) + numberOfHoursForDay > 8;
```

Comment:
Are you sure that we dont need parseInt from `parseInt(numberOfHoursForDay)` ? The propTypes hasn't a restriction as strong as Typescript types.

Reply:
Or maybe, you are supposed to fail if you dont respect the contract of types.

Reply:
The propType for `numberOfHoursForDay` was updated from object to number... this is why I think we don't need a parseInt here 
```js
SelectHoursComponent.propTypes = {
  numberOfHoursForDay: PropTypes.number.isRequired,
  entryDuration: PropTypes.number,
};
```

Reply:
sorry my mistake, forget my last comment